internal/driver/glfw/window_goxjs.go

```diff
@@ -388,8 +311,7 @@ func keyToName(code glfw.Key, scancode int) fyne.KeyName {
 	}
 
 	//	keyName := glfw.GetKeyName(code, scancode)
-	//	ret, ok = keyNameMap[keyName]
-	//	if !ok {
+	// if unknownKey(keyName) {
```

Comment:
Are these commented code remain necessary to stay here?

Reply:
I think they are supposed to be fixed sometime in the future when the web driver gets more features. Am I right, @Bluebugs?

Reply:
Yes, I still need to investigate that code path.